dynawo/sources/Common/DYNTimer.cpp

```diff
@@ -51,6 +51,9 @@ Timers::add(const std::string& name, const double& time) {
 
 void
 Timers::add_(const std::string& name, const double& time) {
+#ifdef LANG_CXX11
+  std::unique_lock<std::mutex> lock(timersMutex_);
```

Comment:
This one is a performance killer (I'm not even sure we could get to the end of a MC in a reasonable time), is there a way to avoid this, for example by using a variant like structure?

Reply:
or by disabling the timers in a MT environment?

Reply:
This add is not done often  and only in debug and even with a variant like structure, we would still a mutex

Reply:
This add is done each time a timer is destroyed, so for example after each evalF, evalJ, evalFAdept of modelica models or network component, after each evalMode, evalG .. in model multi, so it done very often

Reply:
Correct, my bad. But it is done only in debug. Either way, the only other way is transforming the singleton Timers into a multiton by thread

Reply:
Is there a way to simply disable the timers in a MT environment?